REWRITES.md

```diff
@@ -0,0 +1,14 @@
+> This document outlines the changes between the old [Holoitems](https://github.com/dlee13/OldHoloItems) and the rewrite.
+> Sorted Chronologically
+
+# Magnet
+(Please fill in, I haven't been following at this point)
+
+# Tide Rider
+Fixed a bug where players could scale a wall with the Tide Rider.
+
+# Backdash
+Implemented without changes to functionality.
```

Comment:
The intention with the old item was for it to lose durability each time it was used, though it failed at that due to Klin's infinite durability bug. The new item doesn't touch durability.